src/dailies.ts

```diff
@@ -661,7 +664,7 @@ function tomeSummons(): void {
 function extrude(): void {
   if (SourceTerminal.have()) {
     const extrudeConsumables = $items`browser cookie, hacked gibson`;
-    const bestExtrude = extrudeConsumables.sort((a, b) => garboValue(b) - garboValue(a))[0];
+    const bestExtrude = maxBy(extrudeConsumables, garboValue);
```

Comment:
i have a hot take, which is that garbo should either include handling for software bug or a comment noting why we're ignoring software bug. it's reasonable to ignore it from a "slightly annoying to code" sense, since you can only generate 1/day, but people with ungodly amounts of source essence probably could use a few extra for slight extra profits. at least, used to -- it is possible the market on them has collapsed, and garbo would never buy them anyway. still, just a thought here.

Reply:
Gonna declare that out of scope but a neat PR choice for someone

Reply:
Is there mafia tracking?